torch/_dynamo/config.py

```diff
@@ -198,6 +198,9 @@
 # root folder of the project
 base_dir = dirname(dirname(dirname(abspath(__file__))))
 
+# trace through numpy ndarray as tensor and try to translate numpy function to torch function.
+trace_numpy = False
```

Comment:
Can we have a better config name? E.g, numpy_ndarray_as_tensor?

Reply:
Also, NGL, let's figure out how to make this true by default.